mds/models.py

```diff
@@ -39,16 +39,38 @@ class Meta:
         abstract = True
 
 
+def api_authentication_default():
+    """Default value for the API authentication field of a provider.
+
+    Other keys are specific to each type:
+    - token: "header" and "token"
+    - oauth2: "client_id" and "client_secret"
+    """
+    return {"type": "none"}
+
+
+def api_configuration_default():
+    """Default value for the API configuration field of a provider.
+
+    Some provider implementations can be picky, configure these special cases here.
+    """
+    return {
+        "trailing_slash": False,  # Some providers endpoint won't reply without it
+        "bluela_dash": False,  # FIXME Remove when "/statues_changes/" is exposed
+    }
+
+
 class Provider(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4)
     name = UnboundedCharField(default=str)
     logo_b64 = UnboundedCharField(null=True, blank=True, default=None)
     device_category = UnboundedCharField(choices=enums.choices(enums.DEVICE_CATEGORY))
     base_api_url = UnboundedCharField(default="", verbose_name="Base API URL")
-    # Only mandatory key is "type": "none", "token" or "oauth2"
-    # other keys are specific to each type, e.g. client_id and secret for oauth2
-    authentication = pg_fields.HStoreField(
-        default=dict, verbose_name="API Authentication"
+    api_authentication = pg_fields.HStoreField(
```

Comment:
why not JSONField for this one ? (I know I'm late and this was like this before)

Reply:
Now that I have two fields, I seriously consider it. And I guess Postgres can handle the casting just fine (HStore is basically a limited JSON object).

Or since I migrate the old field, just drop it anyway and create a new one.